packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json

```diff
@@ -3,10 +3,6 @@
   "pci_projects_project_creating_secondary_title": "La création de votre projet est en cours",
   "pci_projects_project_creating_explain1": "Pour finaliser la création de votre projet, veuillez effectuer le paiement du bon de commande <a href=\"{{ href }}\">{{ orderId }}</a>.",
   "pci_projects_project_creating_explain2": "Si cette page est toujours présente plusieurs minutes après le paiement du bon de commande, merci de contacter <a href=\"{{ href }}\">notre service client</a>.",
-  "pci_projects_project_creating_not_paid_title": "En route pour l'aventure Public Cloud",
-  "pci_projects_project_creating_not_paid": "Il semble que vous ayez un bon de commande de {{ amount }} de crédits en attente. Nous vous invitons à le régler en cliquant sur \"Finaliser ma commande\" pour lancer la création de votre projet.",
-  "pci_projects_project_creating_not_paid_explain1": "En cliquant sur « Finaliser ma commande » vous allez être redirigé sur un bon de commande qu’il faudra régler par le moyen de paiement que vous aurez choisi.",
```

Comment:
Same for `pci_projects_project_creating_not_paid_explain1`.

Please refer to: https://github.com/ovh-ux/manager/blob/develop/packages/manager/modules/pci/src/projects/project/creating/notPaid/notPaid.html#L36.